common/static/js/vendor/requirejs/text.js

```diff
@@ -139,6 +139,12 @@ var requirejs_text_function = function (module) {
             uPort = uHostName[1];
             uHostName = uHostName[0];
 
+            // Allowing to load static files from our S3 share
+            const allowed_hosts = ['lms-juniper-staticfiles-codeinstitute.s3.eu-west-1.amazonaws.com']
```

Comment:
can we replace this with settings.STATIC_URL?

Reply:
Updated to always fetch from s3 as discussed.